src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -522,6 +523,14 @@ int sd_dhcp_client_set_mtu(sd_dhcp_client *client, uint32_t mtu) {
         return 0;
 }
 
+int sd_dhcp_client_set_max_client_attempt(sd_dhcp_client *client, uint64_t attempt) {
```

Comment:
No need to repeat "client" here either. When the setting name is changed, this should be adjusted to match.